docs/docs/50-user-guide/60-reference-docs/80-webhook-receivers/generic.md

```diff
@@ -4,8 +4,234 @@ sidebar_label: Generic
 
 # Generic Webhook Receiver
 
+The Generic webhook receiver responds to events originating from arbitrary
+repositories by performing `Action`s on `Target`s. Consider the example of a 
+`Warehouse` refresh. In this example, the `Action` is the refresh and the 
+`Target` is the `Warehouse`.
+
+:::note
+A generic webhook receiver is not limited to a single `Action` and `Action`'s 
+are not limited to a single `Target`.
+:::
+
+:::info
+"Refreshing" a `Warehouse` resource means enqueuing it for immediate
+reconciliation by the Kargo controller, which will execute the discovery of new
+artifacts from all repositories to which that `Warehouse` subscribes.
+:::
```

Comment:
These seem they would be sufficiently covered, although less explicitly so, by the suggested modifications to the text at the beginning of the section. For an introductory section, I think that suggestion covers these ideas well enough without putting an overly fine a point on it. Examples will drive home the point that the introductory text teed up.